src/compiler/diagnosticMessages.json

```diff
@@ -195,7 +195,7 @@
         "category": "Error",
         "code": 1063
     },
-    "Ambient enum elements can only have integer literal initializers.": {
+    "In ambient enum declarations member initializer must be constant expression.": {

```

Comment:
Current message is based on the text that we show for const enums:
    "In 'const' enum declarations member initializer must be constant expression.": {